jane-php/open-api-common/6.0/config/packages/jane_open_api.yaml

```diff
@@ -0,0 +1,9 @@
+#services:
+#  _defaults:
+#    autowire: true
+#    autoconfigure: true
+#    public: false
+#
+#  MyApp\Library\Generated\:
+#    resource: '%kernel.root_dir%/generated/*'
```

Comment:
This file only has comments. We dont need that.

Reply:
This is on purpose, as stated in https://github.com/symfony/recipes-contrib/pull/888/files#diff-86fec314f57019f4403b1f6a4e9c30d6R9 or https://github.com/symfony/recipes-contrib/pull/888/files#diff-35c3cdf07a7584253c1adce3da720b28R9, it needs to be edited. I provide this config because it's optimized for Jane